cdap-app-fabric/src/main/java/co/cask/cdap/app/runtime/AbstractProgramRuntimeService.java

```diff
@@ -319,6 +320,27 @@ public boolean checkAnyRunning(Predicate<Id.Program> predicate, ProgramType... t
   }
 
   @Override
+  public Set<Id.Program> getAnyRunning(Predicate<Id.Program> predicate, ProgramType... types) {
```

Comment:
Shouldn't add this method in this class. This method should be in the `ApplicationLifecycleService` instead.